docs/modules/general/pages/bootstrapping.adoc

```diff
@@ -0,0 +1,567 @@
+import { APP_INITIALIZER, DoBootstrap, Injector, NgModule } from '@angular/core'
+import { Router, RouterModule, Routes } from '@angular/router'
+import { BrowserModule } from '@angular/platform-browser'
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
+import { CommonModule } from '@angular/common'
+import { MissingTranslationHandler, TranslateLoader, TranslateModule } from '@ngx-translate/core'
+import {
+  AppStateService,
+  ConfigurationService,
+  createTranslateLoader,
+  PortalCoreModule,
+  PortalMissingTranslationHandler,
+  PortalApiConfiguration,
+} from '@onecx/portal-integration-angular'
+import { AngularAuthModule } from '@onecx/angular-auth'
+import { createAppEntrypoint, initializeRouter, startsWith } from '@onecx/angular-webcomponents'
+import { addInitializeModuleGuard } from '@onecx/angular-integration-interface'
+import { Configuration } from './shared/generated'
+
+@Component({
+  selector: 'app-root',
+  template: `<router-outlet></router-outlet>`
+})
+export class AppEntrypointComponent {}
+
+export const routes: Routes = [
+  {
+    matcher: startsWith(''),
+    loadChildren: () => import('./feature/feature.module').then((mod) => mod.FeatureModule)
+  },
+  {
+    matcher: startsWith('tracking'),
+    loadChildren: () => import('./tracking/tracking.module').then((mod) => mod.TrackingModule)
+  }
+]
+
+function apiConfigProvider(configService: ConfigurationService, appStateService: AppStateService) {
+  return new PortalApiConfiguration(Configuration, environment.apiPrefix, configService, appStateService)
+}
+
+@NgModule({
+    declarations: [AppEntrypointComponent],
+    imports: [
+        CommonModule,
+        PortalCoreModule.forMicroFrontend(),
+        RouterModule.forRoot(addInitializeModuleGuard(routes)),
+        TranslateModule.forRoot({
+        extend: true,
+        isolate: false,
+        loader: {
+            provide: TranslateLoader,
+            useFactory: createTranslateLoader,
+            deps: [HttpClient, AppStateService]
+        },
+        missingTranslationHandler: {
+            provide: MissingTranslationHandler,
+            useClass: PortalMissingTranslationHandler
+        }
+        }),
+        BrowserModule,
+        AngularAuthModule,
+        BrowserAnimationsModule,
+    ],
+    exports: [],
+    providers: [
+        {
+            provide: Configuration,
+            useFactory: apiConfigProvider,
+            deps: [ConfigurationService, AppStateService]
+        },
+        {
+            provide: APP_INITIALIZER,
+            useFactory: initializeRouter,
+            multi: true,
+            deps: [Router, AppStateService]
+        },
+        provideHttpClient(withInterceptorsFromDi())
+    ]
+})
+export class ExampleModule implements DoBootstrap {
+    constructor(private readonly injector: Injector) {}
+
+    ngDoBootstrap(): void {
+        createAppEntrypoint(AppEntrypointComponent, 'example-component', this.injector)
```

Comment:
createAppEntrypoint(AppEntrypointComponent, 'example-**web**(?)component', this.injector)